packages/events/ResponderEventPlugin.js

```diff
@@ -591,6 +627,14 @@ const ResponderEventPlugin = {
     injectGlobalInteractionHandler: function(GlobalInteractionHandler) {
       ResponderEventPlugin.GlobalInteractionHandler = GlobalInteractionHandler;
     },
+
+    /**
+     * @param Object TopLevelTypes
+     * Object that maps top level types to the ID used in the react system.
+     */
+    injectTopLevelTypes: function(TopLevelTypes) {
+      ResponderEventPlugin.TopLevelTypes = TopLevelTypes;
```

Comment:
Instead of injection, could we move this to isntantiation time?

```
createResponderEventPlugin(TopLevelTypes)
```

Reply:
Sure! Should we follow the same pattern for `ResponderTouchHistoryStore`?

```js
createResponderTouchHistoryStore(EventUtils);
```

We can keep it in the closure of `createResponderEventPlugin(TopLevelTypes)` forever since it seems that RNW no longer depends on it (and that appears to have been the only reason why it's added as an export to `ReactDOMUnstableNativeDependencies`).